lua/starfall/libs_cl/render.lua

```diff
@@ -2249,6 +2279,19 @@ function render_library.readPixel(x, y)
 	return setmetatable({render.ReadPixel(x, y)}, col_meta)
 end
 
+--- Reads the color of the specified pixel.
+-- @param number x Pixel x-coordinate.
+-- @param number y Pixel y-coordinate.
+-- @return number The red channel value.
+-- @return number The green channel value.
+-- @return number The blue channel value.
+-- @return number The alpha channel value.
+function render_library.readPixelRGBA(x, y)
+	if not renderdata.isRendering then SF.Throw("Not in rendering hook.", 2) end
+	local r, g, b, a = render.ReadPixel(x, y)
+	return r, g, b, a or 255
```

Comment:
'a' should never be nil should it?

Reply:
https://gmodwiki.com/render.ReadPixel
The alpha channel value or no value if the render target has no alpha channel.

Reply:
All the rendertargets will have alpha